BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -578,7 +578,7 @@ def extract (self, name = '', file_path = ''):
 
             # create header entry
             auth_type_str = self.get_auth_type_str (self.header.auth_type)
-            key_file = 'CONTAINER_KEY_ID' if auth_type_str.startswith('RSA') else ''
+            key_file = 'KEY_ID_CONTAINER_RSA2048' if auth_type_str.startswith('RSA') else ''

```

Comment:
This Id need to be generated based on auth type while doing container extraction. I would improve this as separate patch,